-------------------------------------------------------------------------------
Test set: com.cloudwebshop.remotetests.UserServiceRemoteTest
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 8.801 s <<< FAILURE! -- in com.cloudwebshop.remotetests.UserServiceRemoteTest
com.cloudwebshop.remotetests.UserServiceRemoteTest.testCreateOrderForUser -- Time elapsed: 0.411 s <<< ERROR!
org.springframework.web.client.HttpClientErrorException$NotFound: 404 : "{"errorId":"746d691e-4fd6-4fdc-b56b-6bbde7dad89e","timestamp":"2025-08-27T14:08:38.188787924","status":404,"error":"Not Found","message":"User not found with id: 11111111-1111-1111-1111-111111111111","path":"uri=/api/v1/users/profile"}"
	at org.springframework.web.client.HttpClientErrorException.create(HttpClientErrorException.java:112)
	at org.springframework.web.client.DefaultResponseErrorHandler.handleError(DefaultResponseErrorHandler.java:183)
	at org.springframework.web.client.DefaultResponseErrorHandler.handleError(DefaultResponseErrorHandler.java:137)
	at org.springframework.web.client.ResponseErrorHandler.handleError(ResponseErrorHandler.java:63)
	at org.springframework.web.client.RestTemplate.handleResponse(RestTemplate.java:942)
	at org.springframework.web.client.RestTemplate.doExecute(RestTemplate.java:891)
	at org.springframework.web.client.RestTemplate.execute(RestTemplate.java:790)
	at org.springframework.web.client.RestTemplate.getForEntity(RestTemplate.java:431)
	at com.cloudwebshop.remotetests.UserServiceRemoteTest.testCreateOrderForUser(UserServiceRemoteTest.java:47)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
